internal/testing/test_test.go

```diff
@@ -0,0 +1,94 @@
+package testing
+
+import "testing"
+
+func TestRunTests(t *testing.T) {
+	type args struct {
+		directory   string
+		verbose     bool
+		exitOnError bool
+	}
+	tests := []struct {
```

Comment:
Detail : to be consistent over the whole codebase, don't you prefer to switch to a map ?
As used [here](https://github.com/Joffref/genz/blob/master/internal/parser/parser_test.go#L426)
```suggestion
	tests := map[string]struct {
	   ...
	}
	for name, tc := range tests {
	   t.Run(name, func...)
	}
```